models/models_methods.go

```diff
@@ -864,6 +952,34 @@ func MergeStepWith(step, otherStep stepmanModels.StepModel) (stepmanModels.StepM
 	return step, nil
 }
 
+// ----------------------------
+// --- WorkflowIDData
+
+// GetWorkflowIDWorkflowDataPair ...
+func GetWorkflowIDWorkflowDataPair(workflowListItem WorkflowListItemModel) (string, WorkflowModel, error) {
```

Comment:
Where are we using  the returned `WorkflowModel`? I've could not found usage for it.

Reply:
We don't use it for nothing yet, it's a replica of `GetStepIDStepDataPair`. We might want to use it later, but we can remove returning the model for now.

Reply:
Ok, in this case please remove both unused values.

Reply:
Ok, updated the PR.